src/SlackFunction.ts

```diff
@@ -46,83 +113,412 @@ export class SlackFunction {
   private callbackId: string;
 
   /**
-   * @description fn to to process corresponding
+   * @description handler to to process corresponding
```

Comment:
Deno Slack SDK is using the terminology "handlers" to I am being consistent with them.